### PR TITLE
Feature/icon package

### PR DESCRIPTION
**Pull request by andrew-boyd**
Allows assignment of icons to inputs via `icon`, `icon-prefix` and `icon-suffix` props. Currently in draft as we await cleaned up icons for the built-in icon set.

### CODE REVIEW

**Comment by andrew-boyd** [comment]
@justin-schroeder I've ported over the newly delivered icons — I'm going to start a complimentary docs branch.

**Comment by justin-schroeder** [comment]
@andrew-boyd mind checking out these failing tests?

**Comment by andrew-boyd** [comment]
> @andrew-boyd mind checking out these failing tests?

Looks like it was a file casing issue, resolved now.

File: packages/themes/src/tailwindcss/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const FormKitVariants = plugin(({ addVariant, theme }) => {
 
   addVariant('formkit-action', ['.formkit-actions &', '.formkit-actions&']);
 
-  ['disabled', 'invalid', 'errors', 'complete', 'loading', 'submitted', 'multiple', ...attributes].forEach((attribute) => {
+  ['disabled', 'invalid', 'errors', 'complete', 'loading', 'submitted', 'multiple', 'has-prefix-icon', 'has-suffix-icon', ...attributes].forEach((attribute) => {
```
Comment (justin-schroeder) [inline packages/themes/src/tailwindcss/index.ts:13]: nice! I dont understand how this works exactly, but I get what we're providing and its cool

File: packages/icons/__tests__/icons.spec.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import { createIconPlugin, inputIcons, ethereum, apple } from '../src'
+import { createNode, getNode, FormKitNode } from '@formkit/core'
+import { plugin as vuePlugin, FormKit, defaultConfig } from '@formkit/vue'
+import { mount } from '@vue/test-utils'
+import { jest } from '@jest/globals'
+
```
Comment (justin-schroeder) [inline packages/icons/__tests__/icons.spec.ts:6]: These look like great tests to me.

File: packages/icons/__tests__/icons.spec.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      global: {
+        plugins: [
+          [vuePlugin, defaultConfig({
+            plugins: [iconPlugin]
+          })]
+        ]
+      }
+    })
+    wrapper.find('.formkit-icon.formkit-prefix').trigger('click')
+    expect(prefixClickHandler).toHaveBeenCalledTimes(1)
```
Comment (justin-schroeder) [inline packages/icons/__tests__/icons.spec.ts:99]: Would be useful to use something like `toHaveBeenCalledNthWith()` here to test that you're getting the right prefix/suffix argument. Not a huge deal.

**Review by justin-schroeder — APPROVED** [review]
Looks good to me. Had one suggestion on a test, but feel free to merge and publish on @next when you feel its good.

**Pull request merged**